src/renderers/webgl/WebGLPrograms.js

```diff
@@ -184,6 +185,10 @@ function WebGLPrograms( renderer, cubemaps, cubeuvmaps, extensions, capabilities
 			clearcoatRoughnessMap: useClearcoat && !! material.clearcoatRoughnessMap,
 			clearcoatNormalMap: useClearcoat && !! material.clearcoatNormalMap,
 
+			iridescence: useIridescence,
+			iridescenceMap: useIridescence && !! material.iridescenceMap,
+			iridescenceThicknessMap: useIridescence && !! material.iridescenceThicknessMap,
```

Comment:
You have to honor these new program parameters in `getProgramCacheKeyBooleans()`.

Reply:
@Mugen87 done ✅